bayesflow/inference_networks.py

```diff
@@ -187,8 +189,8 @@ def forward(self, targets, condition, **kwargs):
         # not in call(), since methods are public
         if self.soft_flow and condition is not None:
             # Extract shapes of tensors
-            target_shape = tf.shape(targets)
-            condition_shape = tf.shape(condition)
+            target_shape = keras.backend.shape(targets)
+            condition_shape = keras.backend.shape(condition)
```

Comment:
as discussed, please use `keras.ops`

Reply:
Changed requested `keras.backend` instances to `keras.ops` in `inference_networks.py`. (see [47fbbbd](https://github.com/stefanradev93/BayesFlow/pull/165/commits/47fbbbd4acb22c391b483c845f73e6862094c07a))
Will change remaining `keras.backend` to `keras.ops` calls with next update

Reply:
Remaining `keras.backend` calls switched to respective `keras.ops` functions (see [b9d5f16](https://github.com/stefanradev93/BayesFlow/pull/165/commits/b9d5f16a9e2a9b80db03f7d8262a4b39d15e2f8c))